startleft/cli.py

```diff
@@ -129,9 +118,15 @@ def threatmodel(recreate, irius_server, api_token, filename):
     """
     Uploads an OTM file to IriusRisk
     """
-    otm_to_ir = OtmToIr(irius_server, api_token)
-    logger.info("Uploading OTM file and generating the IriusRisk threat model")
-    otm_to_ir.run(recreate, filename)
+    otm_project = OtmProject.from_otm_file(filename)
+    otm_service = OtmProjectService(IriusriskProjectRepository(api_token, irius_server))
```

Comment:
From this line till the end of this cli command, the code is exactly the same as in **_run_** cli command. Could be a good idea to create a common method for that.